### ISSUE TITLE
Why there is no positional embeds in the flow decoder transformer layers inputs?

### ISSUE THREAD

**Issue by JohnHerry** [FunAudioLLM/CosyVoice#381]
the flow model in cosyvoice, its encoder Conformers contains position embeds while in its decoder transformers, I see no such addition. is that means no benifit  here in flow-matching? sorry for it if I did not find the code.

**Comment by aluminumbox** [comment]
well decoder is from matha-tts, we haven't tested whether position embedding can help in decoder, maybe it can

**Comment by JohnHerry** [comment]
Yes there is no position in macha, may be it assume the intput hidden had contain such info?
